assignments/lambda-classes.js

```diff
@@ -1 +1,152 @@
-// CODE here for your Lambda Classes
+// CODE here for your Lambda Classes// Step 1
+class Person {
+  constructor(attributes) {
+    console.log(attributes);
+    console.log(this)
+    this.name = attributes.name;
+    this.age = attributes.age;
+    this.location = attributes.location;
+  }
+
+  speak() {
+    return `Hello my name is ${this.name}, I am from ${this.location}`
+  }
+}
+
+//Person
+const fred = new Person({
+  age: 35,
+  name: 'Fred',
+  location: 'RockBed'
+});
+
+const willma = new Person({
+  age: 32,
+  name: 'Willma',
+  location: 'RockBed'
+});
+
+console.log(fred)
+console.log(fred.speak())
+
+class Instructor extends Person {
+  constructor(attributes) {
+    super(attributes)
+    console.log(attributes);
+    console.log(this)
+    this.name = attributes.name;
+    this.age = attributes.age;
+    this.location = attributes.location;
```

Comment:
Same here, you do not need to add `name`, `age` and `location` attributes to class `Instructor` since it inherits the attributes and methods of class `Person` with the use of `super()`